packages/astro/src/content/data-store.ts

```diff
@@ -371,5 +470,13 @@ function dataStoreSingleton() {
 	};
 }
 
+// TODO: find a better place to put this image
+export function contentModuleToId(fileName: string, specifier: string) {
```

Comment:
I don't think we need to have specifier as an argument. Just hard code it to `DEFERRED_MODULE`